src/context/product-context.jsx

```diff
@@ -0,0 +1,22 @@
+import { createContext, useContext, useReducer } from "react";
+import { ProductReducer } from "../reducers/product-reducer";
+
+const ProductContext = createContext();
+
+const ProductProvider = ({ children }) => {
+	const [productState, dispatch] = useReducer(ProductReducer, {
```

Comment:
maybe name it productDispatch or something as when you will proceed further there will be dispatch functions for other functionalities as well

Reply:
yeah makes sense sakshi , i'll change that